### PR TITLE
Track floating deps at master

### PR DESCRIPTION
**Pull request by navidshaikh**
## Description
- Track floating deps at master (`./hack/update-deps.sh --upgrade`)

## Changes
- Update kn version command to print correct version of eventing/serving dep
```
Version:      v20200715-local-98c38a60-dirty
Build Date:   2020-07-15 08:06:47
Git Revision: 98c38a60
Supported APIs:
* Serving
  - serving.knative.dev/v1 (knative-serving v0.16.1-0.20200715073232-81d40bfc82a6)
* Eventing
  - sources.knative.dev/v1alpha2 (knative-eventing v0.16.1-0.20200715062032-28f9f61e6131)
  - eventing.knative.dev/v1beta1 (knative-eventing v0.16.1-0.20200715062032-28f9f61e6131)
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,19 +5,19 @@ go 1.14
 require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v0.0.6
+	github.com/spf13/cobra v1.0.0
```
Comment (navidshaikh) [inline go.mod:8]: @rhuss : I'll raise another PR to pin cobra at master with entry for `replace` section removed.

Comment (rhuss) [reply go.mod:8]: 👍

**Pull request merged**